wondrous-app/components/Settings/TaskImport/GenericImportTaskModal/styles.tsx

```diff
@@ -0,0 +1,290 @@
+  font-family: ${typography.fontFamily};
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: 1%;
+  color: ${palette.blue20};
+  width: max-content;
+`;
+
+export const GenericImportTaskModalLabelHelperLink = styled.a`
+  color: ${palette.highlightBlue};
+  margin-left: 6px;
+`;
+
+export const GenericImportTaskModalError = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    color: ${palette.red400};
+    padding-left: 4px;
+  }
+`;
+
+const StyledSelect = styled(Select)`
+  && {
+    background: ${palette.background.default};
+    color: ${palette.white};
+    height: 42px;
+    width: 100%;
+    border-radius: 6px;
+    border: 1px solid transparent;
+    transition: border 0.2s ease-out;
+
+    :hover {
+      border: 1px solid ${palette.grey79};
+    }
+
+    svg {
+      color: ${(props) => (props.isActive ? palette.white : palette.grey58)};
+      transition: transform 0.2s ease-out;
+    }
+
+    p {
+      margin: 0;
+    }
+  }
+`;
+
+export const GenericImportTaskModalSelect = styled(({ className, ...props }) => (
+  <StyledSelect
+    {...props}
+    {...className}
+    MenuProps={{ classes: { paper: className } }}
+    isActive={!!props.value?.value}
+  />
+))`
+  &.MuiPaper-root {
+    background: ${palette.grey77};
+    width: 100%;
+    max-width: 510px;
+    color: ${palette.white};
+  }
+
+  &.MuiPaper-root > .MuiList-root {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+  }
+
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 4px;
+  }
+`;
+
+export const GenericImportTaskModalSelectMenuItem = styled(MenuItem)`
+  && {
+    background: ${palette.grey87};
+    color: ${palette.white};
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    padding: 10px;
+    transition: background 0.2s ease-out;
+
+    :hover {
+      background: ${palette.black92};
+    }
+  }
+`;
+
+export const GenericImportTaskModalSelectValueDisplay = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+export const GenericImportTaskModalSelectValueDisplayText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 19.14px;
+    color: ${(props) => (props.isActive ? palette.white : palette.grey57)};
+  }
+`;
+
+export const GenericImportTaskModalBodyExpandedViewWrapper = styled(Accordion)`
+  && {
+    background: ${palette.grey900};
+    margin-top: ${(props) => (props.expanded ? 0 : '-24px')};
+
+    & .MuiAccordion-region {
+      display: flex;
+      flex-direction: column;
+      gap: 26px;
+      box-shadow: none;
+    }
+
+    &.MuiPaper-root.MuiAccordion-root {
+      box-shadow: none;
+    }
+
+    &.MuiPaper-root::before {
+      display: none;
+    }
+  }
+`;
+
+export const GenericImportTaskModalBodyExpandedViewInvisibleState = styled(AccordionSummary)`
```

Comment:
use AccordionSummary and `sx`

Reply:
Seems like AccordionSummary doesn't support display prop

Reply:
I've not tried for this component, but from docs I see that it's supported